build_all.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eux
+
+for pb in $(find packagingbuild -name Dockerfile); do
+	path=$(dirname $pb)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/packagingbuild:$flavor .) || exit -1
+done
+
+for pt in $(find packagingtest -name Dockerfile); do
+	path=$(dirname $pt)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/packagingtest:${flavor}-systemd -t stackstorm/packagingtest:${flavor} .) || exit -1
```

Comment:
```suggestion
	(cd $path; docker build -t stackstorm/packagingtest:${flavor}-systemd .) || exit -1
```
Please keep ony the first tag. The images should have init system included in the Docker tag by a reason.

Shipping the Docker images with the init system is unobvious and should always be signalized.
Easy way of highlighting that is with tag.

We shouldn't be hiding magic facts like that just because it looks nicer.